bindings/pyroot/pythonizations/python/ROOT/pythonization/_tfile.py

```diff
@@ -45,6 +111,9 @@ def _TFileOpen(klass, *args):
 # Pythonizor function
 @pythonization()
 def pythonize_tfile(klass, name):
+    # TFile inherits from
```

Comment:
I would use the `"""` string block to mark it as a doc string of the function. But mainly my opinion, no technical reason ;)